inference-engine/ie_bridges/java/tests/README.md

```diff
@@ -0,0 +1,30 @@
+## Build
+Before building download ```junit-x.xx.jar```:
+https://search.maven.org/search?q=g:junit%20AND%20a:junit
+
+and clone the repository with models:
+```bash
+git clone https://github.com/openvinotoolkit/testdata
+git lfs pull
+```
+
+Create an environment variable with Inference Engine installation path:
+```bash
+export IE_PATH=/path/to/dldt/build
+```
+
+To compile tests use:
+```bash
+javac -cp ".:/path_to_junit/junit-x.xx.jar:${IE_PATH}/inference-engine/ie_bridges/java/inference_engine_java_api.jar" *.java
```

Comment:
Can we make tests compilation during CMake step? If `ENABLE_JAVA` and `ENABLE_TESTS` are `ON`